src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -88,7 +88,8 @@ private ExpirationTool() {
 
     String where = format("WHERE " +
         "(jsonb->>'status' = '%1$s' AND jsonb->>'requestExpirationDate' < '%3$s') OR " +
-        "(jsonb->>'status' = '%2$s' AND jsonb->>'holdShelfExpirationDate' < '%3$s')",
+        "(jsonb->>'status' = '%2$s' AND jsonb->>'holdShelfExpirationDate' < '%3$s') " +
+        "LIMIT 150",
```

Comment:
Why has this limit been introduced?

Reply:
To process limited amount of requests in one run, prevents potential overlaps between runs

Reply:
How was 150 decided upon as an appropriate number?

Reply:
150 gives us 1050 processed requests every quarter of an hour.
Tested locally with 10 requests queues * 150, average process time <10s.
Still request queue size is unpredictable, there are only average values available.
So i've included potential processing delays, possible long queues etc.

Reply:
Fair enough